source/_integrations/motion_blinds.markdown

```diff
@@ -0,0 +1,41 @@
+---
+title: Motion Blinds
+description: Instructions on how to integrate Motion Blinds from Coulisse B.V. into Home Assistant.
+ha_category:
+  - Cover
+ha_iot_class: Local Polling
+ha_release: 0.119.0
+ha_domain: motion_blinds
+ha_codeowners:
+  - '@starkillerOG'
+ha_config_flow: true
+---
+
+The `motion_blinds` platform allows you to control [Motion Blinds](https://motion-blinds.com) from [Coulisse B.V.](https://coulisse.com/products/motion) within Home Assistant.
+
+To add a Motion Bridge to your installation, click Configuration in the sidebar, then click Integrations. Click the + icon in the lower right. Then search for "Motion Blinds" and enter the setup.
+
+{% configuration %}
+host:
+  description: IP address of the Motion Bridge, e.g., 192.168.1.100.
+  required: true
+  type: string
+API Key:
+  description: The API Key of your Motion Bridge, see [Retrieving the API Key](#retrieving-the-api-key).
+  required: true
+  type: string
+{% endconfiguration %}
```

Comment:
This is used for YAML configuration entries.

Reply:
I removed this and rephrased the important information.